playbooks/prepare_vm_for_minishift.yml

```diff
@@ -0,0 +1,48 @@
+- hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("yes") }}'
+  gather_facts: '{{ gather_facts | default("no") }}'
+  vars:
+    setup_nested_virt: false
+  tasks:
+    - name: "Install python if not installed"
+      raw: test -e /usr/bin/python || sudo yum install -y python
+    - name: "Install docker, net-tools and firewalld"
+      package:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - docker
+        - net-tools
+        - firewalld
+    - name: "Ensure group docker exists"
+      group:
+        name: docker
+        state: present
+    - name: "Adding existing user {{ ansible_user }} to group docker"
+      user:
+        name: "{{ ansible_user }}"
+        groups: docker
+        append: yes
+    - name: "Ensure that docker service is running"
+      service:
+        name: docker
+        state: reloaded
```

Comment:
Idk if reload alone will start the service, what if it's not started before this task, it won't restart imo. Maybe we are better off using started or use a debug task? Except this, rest lgtm.

Reply:
The reloaded state starts the service if it's not already started.

Reply:
then we good, +2